plugins/indexing/oracle-program/module.go

```diff
@@ -0,0 +1,58 @@
+package oracleprogram
+
+import (
+	"bytes"
+	"encoding/hex"
+	"time"
+
+	storetypes "cosmossdk.io/store/types"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+
+	"github.com/sedaprotocol/seda-chain/plugins/indexing/log"
+	"github.com/sedaprotocol/seda-chain/plugins/indexing/types"
+	oracleprogramtypes "github.com/sedaprotocol/seda-chain/x/wasm-storage/types"
```

Comment:
Wouldn't it be more appropriate to continue calling it `wasmstoragetypes`? Are we planning to change the name `DataRequestWasm` to `OracleProgram`?

Reply:
That’s how I understood it yeah, basically all WASM/binary stuff becomes Oracle Program.